operations/runbooks.rst

```diff
@@ -96,3 +159,95 @@ Zápis z jednorázového elektronického hlasování
    Pokud nějaký člen rady nehlasoval, tak není potřeba jej vůbec uvádět.
 
 #. Pošli Pull Request s touto změnou. Před jeho přijetím by měl být schválen alespoň jedním dalším členem výboru (můžeš nastavit tým `@pyvec/board <https://github.com/orgs/pyvec/teams/board>`__ v *reviewers*).
+
+
+.. _navody-pokladnik:
+
+Návody pro pokladníka
+---------------------
+
+.. _proplaceni:
+
+Proplácení účetních dokladů
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
++------------------------------+------------------------------------------------------+
+| **Pro koho je tento návod:** | |:moneybag:| :term:`pokladník <Pokladník>`, předseda |
++------------------------------+------------------------------------------------------+
+
+#. Někdo využívá :ref:`finančních služeb Pyvce <financni-sluzby>` a krok za krokem následuje návod :ref:`jak-proplatit`. Na Pyvec přijde účetní doklad k proplacení a e-mail s dalšími detaily:
+
+   - pod jakou **skupinu** výdaj spadá
+   - **co to je** za věc a **k čemu slouží**
+   - na jaké **číslo účtu** mají být proplaceny peníze
+
+#. Ověříš, že se něco takového projednávalo a schválilo. Ověříš, jestli sedí částky.
+#. Přihlásíš se do `Fio banky <https://ib.fio.cz/ib/login>`_ a odešleš platbu, přičemž využiješ detaily z e-mailu:
+
+   -  číslo účtu
+   -  do zprávy napíšeš:
+
+      .. code-block:: text
+
+         SKUPINA: proplacení účtu za VĚC
+
+#. Propiskou napíšeš na papírový doklad slovo **PYVEC** aby se odlišil od dokladů firmy `Fragaria <https://fragaria.cz/>`_ a dáš jej do šuplíku číslo 2 ve `Fragarii <https://fragaria.cz/>`_, odkud se doklady automagicky odesílají k :term:`účetní <Účetní>`.
+
+.. note::
+
+   Ideálně je každý účetní doklad vyřízen samostatnou platbou. Cílem je, aby :term:`účetní <Účetní>` mohla z výpisu snadno evidovat, co k čemu patří.
+
+   Pokud je toho víc, třeba při proplácení účtenek z `PyCon CZ <https://cz.pycon.org>`_, sloučíš platby podle čísla účtu, kam se mají proplatit peníze, poznamenáš to do zprávy k platbě a papírové doklady se sepnou sponkou s poznámkou, že byly proplaceny dohromady.
+
+
+Vystavení faktury
+^^^^^^^^^^^^^^^^^
+
++------------------------------+------------------------------------------------------+
+| **Pro koho je tento návod:** | |:moneybag:| :term:`pokladník <Pokladník>`, předseda |
++------------------------------+------------------------------------------------------+
+
+#. Někdo Pyvci oznámí "domluvila jsem, že nám `PSF <https://www.python.org/psf/>`_ pošle milion korun jako sponzorský dar"
+#. Zjisti:
+
+   - pod jakou **skupinu** příjem spadá (PyLadies, PyCon CZ, Pyvo v Brně, ...)
```

Comment:
```suggestion
   - pod jakou **skupinu** příjem spadá (PyLadies, PyCon CZ, Pyvo v Brně...)
```

Reply:
? tohle jsi předtím opravoval naopak

Reply:
Tak to jsem asi blázen. Mea culpa. Před výpustku čárka nepatří.

Reply:
https://github.com/pyvec/docs.pyvec.org/pull/98#discussion_r356549671